static/mfb/mfb.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * Material floating button
+ * By: Nobita
+ * Repo and docs: https://github.com/nobitagit/material-floating-button
+ *
+ * License: MIT
+ */
+
+ // build script hook - don't remove
+ ;(function ( window, document, undefined ) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['undefined' is defined but never used. (no-unused-vars)](https://app.codacy.com/manual/m-kozak/cgm-remote-monitor/pullRequest?prid=1196151)